harvard-lts/epadd-curator-app
# Performance Testing

**Adds API calls for performance testing and sets up movement of data from the store to the dropbox for testing**
* * *

**Ticket Link**: https://jira.huit.harvard.edu/browse/LTSEPADD-25

# How should this be tested?
This is on dev currently.  You can test one run through by calling:
http://ltsds-cloud-dev-1.lib.harvard.edu:10586/runTest/EASProjectEmail

This will add data to the S3 bucket.  You can see the data by getting the credentials from LPE for epadd dev and calling:
`aws s3 ls s3://epadd-export-dev/`
You will see `EASProjectEmail` under this

It will be picked up by the cron job every 5 minutes and deposited into the dev dropbox with an osn_<timestamp> name.  This can viewed in` tsds-cloud-dev-1.lib.harvard.edu:/drs2dev/drsfs/dropbox/epadddev_secure/incoming/`

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? yes
- integration tests? this ticket is made for integration testing
